builtin/staker/validation/service.go

```diff
@@ -469,3 +480,7 @@ func makeRewardKey(validator thor.Address, stakingPeriod uint32) thor.Bytes32 {
 	copy(key[4:], validator.Bytes())
 	return key
 }
+
+func (s *Service) ContractAddress() thor.Address {
```

Comment:
This is not necessary, the address is passed from the constructor `Staker.New(address`. Normally, it's `builtin.Staker.Address`.